requirements.txt

```diff
@@ -124,7 +127,7 @@ pycparser==2.19
 pycryptodome==3.9.0
 pycryptodomex==3.9.0
 PyGithub==1.43.8
-pygpgme==0.3
+# gpgme-devel not available on centos8 pygpgme==0.3
```

Comment:
Even though Centos8 doesn't provide a package (natively?) we should review with @josephschorr to see if this package is required for Quay's functionality. We could always compile it or what-not. I don't think we should let Centos8 block us here.

Reply:
It is required afaik